Common/UnitDefinitions/Volume.json

```diff
@@ -242,7 +242,7 @@
       "PluralName": "UsGallons",
       "FromUnitToBaseFunc": "x*0.00378541",
       "FromBaseToUnitFunc": "x/0.00378541",
-      "Prefixes": [ "Kilo", "Mega" ],
+      "Prefixes": [ "Deca", "Deci", "Hecto", "Kilo", "Mega" ],
```

Comment:
You mentioned reports using 'cGal'/'100 Gal' and 'kGal'/'1000 Gal'.

Please note:
1. The abbreviations used by `ToString()` will in this PR will be `dgal (U.S.)` and `hgal (U.S.)` etc. 
2. `Volume.Parse("5 cGal")` will throw an exception, because parsing is currently case sensitive.

If you really want `ToString()` and `Parse()` to support using uppercase `Gal`, then you need to add that `Gal (U.S.)` abbreviation to the end of the `Abbreviations` array below. It would still need the `(U.S.)` suffix to be consistent with our naming conventions though..

To use a specific abbreviation in `ToString()`, take a look at [custom string formatting](https://github.com/angularsen/UnitsNet/wiki/String-Formatting).

If you are fine with the existing lowercase `gal (U.S.)` abbreviation, then I recommend not adding `Gal (U.S.)` unless it will actually be used.

Reply:
#781 is now merged, so this is point is no longer relevant :-) This PR is then good to go!

Reply:
@angularsen Sorry for delay with my reply. You see, in our system exists a few different names for the same unit. First company names it "kGal", the second - "100 Gal", the third - "Hundred Gallons" and so on. Thus I don't think that there is any standard name of those units in the States, and if there is actually one, looks like no one really respects it. So I would leave it as it is now. :-)